BSP/STM32F413/Src/BSP_Lights.c

```diff
@@ -11,32 +38,21 @@
 void BSP_Lights_Init(void) {
     GPIO_InitTypeDef GPIO_InitStruct;
 	
-	// PC0 : Over current
-	// PC1 : Over temp
-	// PC2 : Over volt
-	// PC3 : Under volt
 	RCC_AHB1PeriphClockCmd(RCC_AHB1Periph_GPIOC, ENABLE);
-	GPIO_InitStruct.GPIO_Pin = GPIO_Pin_0 | GPIO_Pin_1 | GPIO_Pin_2 | GPIO_Pin_3;
-	GPIO_InitStruct.GPIO_Mode = GPIO_Mode_OUT;
-	GPIO_InitStruct.GPIO_OType = GPIO_OType_PP;
-	GPIO_InitStruct.GPIO_PuPd = GPIO_PuPd_DOWN;
+	GPIO_InitStruct.GPIO_Pin = GPIO_Pin_0 | GPIO_Pin_1 | GPIO_Pin_2 | GPIO_Pin_3 | GPIO_Pin_4; 
+	GPIO_InitStruct.GPIO_Mode =  GPIO_Mode_OUT;
+	GPIO_InitStruct.GPIO_OType = GPIO_OType_PP;	
+	GPIO_InitStruct.GPIO_PuPd =  GPIO_PuPd_NOPULL;
 	GPIO_InitStruct.GPIO_Speed = GPIO_Speed_50MHz;
 	GPIO_Init(GPIOC, &GPIO_InitStruct);
-	
-	// PA0 : Run
-	// PA5 : Extra
-	// PA6 : CAN
-	// PA7 : WDog
+
```

Comment:
You know, since we have the arrays of pin numbers and ports anyways, I feel like we could write a more general initialization function.

Certainly not necessary though.